inginious/frontend/plugins/manual_scoring/pages/templates/user_submissions.html

```diff
@@ -0,0 +1,67 @@
+$def with (course, data, task, task_name, username, name, url)
+
+$#
+$# This file is part of UNCode. See the LICENSE and the COPYRIGHTS files for
+$# more information about the licensing of this file.
+$#
+
+
+$var title: $:course.get_name(user_manager.session_language()) - Rubric Scoring
+
+$var Column: $:template_helper.call('course_admin_menu',course=course,current='manual_scoring')
+$ is_admin = user_manager.has_admin_rights_on_course(course)
+
+
+$def NavbarF():
+    <ol class="nav navbar-nav nav-breadcrumb">
+        <li><a href="$get_homepath()/course/$course.get_id()">$course.get_name(user_manager.session_language())</a></li>
+        <li><a href="$get_homepath()/admin/$course.get_id()/manual_scoring"><i class="fa fa-table mr-1"></i>$:_("Rubric Scoring")<span class="sr-only">(current)</span></a></li>
```

Comment:
Is this current too?

```suggestion
        <li><a href="$get_homepath()/admin/$course.get_id()/manual_scoring"><i class="fa fa-table mr-1"></i>$:_("Manual Scoring")<span class="sr-only">(current)</span></a></li>
```